assets/script/script.js

```diff
@@ -39,22 +56,27 @@ function parseLastFMTracksResponse(responseFM) {
     console.log(`artistName: ${artistName}, trackName: ${trackName}`);
     let liEl = $("<li>");
     liEl.text(artistName + " - " + trackName);
-    $("#weather-music-list").append(liEl);
+    $("#player").append(liEl);
```

Comment:
This going away with a change i made - no longer need LastFM. Are you trying to append the artist name here? We did away with that, so this section isn't needed.